Dockerfile.native

```diff
@@ -15,13 +15,14 @@ RUN apt-get update \
 ENV DEBIAN_FRONTEND=dialog
 
 COPY Cargo.* ./
-COPY .cargo ./.cargo
+
```

Comment:
If we do not copy .cargo in here it'll not build as `cpu-target=native` which was the intent of the secondary image.

Reply:
good catch! fixed it now

Reply:
👍